samples/doc_writer_sample_output.py

```diff
@@ -487,7 +499,7 @@
                                 "@id": "updateclass",
                                 "@type": "http://schema.org/UpdateAction",
                                 "description": "null",
-                                "expects": "vocab:singleClass",
+                                "expects": "https://hydrus.com/api/singleClass",
```

Comment:
We don't use the endpoint URI here, this should lead to the definition of what it expects. This would be 'vocab:singleClass'.